staff/pedro-garriganogues/gamma-template/contact.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Contact Us</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="style.css" />
+    <script src="main.js"></script>
+</head>
+
+
+
+<body>
+
+    <header class="header">
+        <h1 class="green-font">Gamma</h1>
+        <img class="burger" src="./images/bars-solid.svg" alt="404">
+        <section class="redirect">
+            <a class="link" href="index.html">Home</a>
+            <a class="link" href="about.html">About</a>
+            <a class="link" href="contact.html">Contact</a>
+        </section>
+
+    </header>
+    <h1 class="green-top white-font" >Contact Us</h1>
+<main class="supermain">
+    <section class="contact">
+        <form action="">
+
```

Comment:
This form tag is empty. It should have the inputs inside

Reply:
Done!